controls/ssl/ssl_certificate.sp

```diff
@@ -0,0 +1,338 @@
+benchmark "ssl_certificate_best_practices" {
+  title         = "SSL/TLS Certificate Best Practices"
+  description   = "Best practices for your domain certificates."
+  documentation = file("./controls/docs/ssl/ssl_certificate_overview.md")
+
+  children = [
+    control.ssl_certificate_domain_name_mismatch,
+    control.ssl_certificate_valid,
+    control.ssl_certificate_not_expired,
+    control.ssl_certificate_not_self_signed,
+    control.ssl_certificate_not_revoked,
+    control.ssl_certificate_secure_private_key,
+    control.ssl_certificate_multiple_hostname,
+    control.ssl_certificate_check_for_reliable_ca,
+    control.ssl_certificate_no_insecure_signature,
+    control.ssl_certificate_transparent,
+    control.ssl_certificate_caa_record_configured,
+  ]
+
+  tags = merge(local.ssl_best_practices_common_tags, {
+    type = "Benchmark"
+  })
+}
+
+control "ssl_certificate_domain_name_mismatch" {
+  title       = "Certificate common name should be listed in subject alternative name (SAN)"
```

Comment:
```suggestion
  title       = "Certificate common names should be listed in subject alternative name (SAN)"
```

Reply:
We usually use plural resource types/names in our titles (since there can be more than 1 row in the results)